platform/qemu.go

```diff
@@ -293,7 +296,28 @@ func mkpath(basedir string) (string, error) {
 	return f.Name(), nil
 }
 
-func CreateQEMUCommand(board, uuid, biosImage, consolePath, confPath, diskImagePath string, isIgnition bool, options MachineOptions) ([]string, []*os.File, error) {
+func CreateOvmfVarsCopy(dir, ovmfVars string) (string, error) {
+	ovmfVarsDst := path.Join(dir, path.Base(ovmfVars))
+	ovmfVarsSrc, err := os.Open(ovmfVars)
+	if err != nil {
+		return "", err
+	}
+	defer ovmfVarsSrc.Close()
+
+	ovmfVarsCopy, err := os.Create(ovmfVarsDst)
+	if err != nil {
+		return "", err
+	}
+	defer ovmfVarsCopy.Close()
+
+	if _, err := io.Copy(ovmfVarsCopy, ovmfVarsSrc); err != nil {
+		os.Remove(ovmfVarsCopy.Name())
+		return "", err
```

Comment:
There is a `CopyRegularFile` function in `github.com/flatcar/mantle/system` package. Maybe you could use it instead?

Reply:
i was looking for something like this. Switched to `CopyRegularFile`